ios/testmanagerd/xcuitestrunner.go

```diff
@@ -249,6 +250,59 @@ type TestConfig struct {
 	Listener *TestListener
 }
 
+func StartXCTestWithConfig(ctx context.Context, xctestrunFilePath string, device ios.DeviceEntry, listener *TestListener) ([]TestSuite, error) {
+	// Parse the .xctestrun file to get the necessary details for TestConfig
+	codec := NewXCTestRunCodec()
+	result, err := codec.ParseFile(xctestrunFilePath)
+	if err != nil {
+		log.Errorf("Error parsing xctestrun file: %v", err)
+		return nil, err
+	}
+
+	testsToRunArg := result.RunnerTests.OnlyTestIdentifiers
+	var testsToRun []string
+	if testsToRunArg != nil && len(testsToRunArg) > 0 {
+		testsToRun = testsToRunArg
+	}
+
+	testsToSkipArg := result.RunnerTests.SkipTestIdentifiers
+	var testsToSkip []string
+	testsToSkip = nil
+	if testsToSkipArg != nil && len(testsToSkipArg) > 0 {
+		testsToSkip = testsToSkipArg
+	}
```

Comment:
we should be able to simplify this to: 
`testsToSkip := result.RunnerTests.SkipTestIdentifiers`

Reply:
done